libraries/loader/src/loader.js

```diff
@@ -2,7 +2,7 @@ const util = require('util');
 const assume = require('assume');
 const TopoSort = require('topo-sort');
 const debug = require('debug')('taskcluster-lib-loader');
-
+var myLoader;//variable to store method load() so that er can use it globally
```

Comment:
We usually prefer to use `let` or `const` for declaring variables.  Is there a reason to pick var?

I'm not sure the reason for storing a module level reference to a load function?  The load function is meant to be returned as a closure, so storing a global reference to the last closure created by the module could have implications that I'm not totally sure of.

Reply:
Will change to `let` . My bad Sorry!